packages/dragdrop/src/index.ts

```diff
@@ -837,30 +837,51 @@ namespace Private {
   }
 
   /**
-   * Find the event target using pointer position.
+   * Find the event target using pointer position if given, or otherwise
+   * the central position of the backdrop.
    */
   export function findElementBehidBackdrop(
-    event: PointerEvent,
+    event?: PointerEvent,
     root: Document | ShadowRoot = document
   ) {
```

Comment:
should we also add 

```ts
export function findElementBehindBackdrop
```

without typo in the name and deprecate this name?

Reply:
Good catch. This is in the `Private` namespace, so I think we can skip deprecation, is my reasoning correct?

Reply:
indeed I miss that - less deprecation :100: